clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -40,8 +40,14 @@
 #include <limits.h>
 #include <stddef.h>
 
+#include <algorithm>
 #include <atomic>
 #include <cstdint>
+#include <iostream>
```

Comment:
See comment below on avoiding these external libraries in online tracer code.

Reply:
Done.